src/test/regress/expected/multi_hash_pruning.out

```diff
@@ -199,6 +199,8 @@ SELECT count(*) FROM orders_hash_partitioned
 	WHERE o_orderkey = 1 OR o_orderkey = 2;
 DEBUG:  predicate pruning for shardId 630001
 DEBUG:  predicate pruning for shardId 630002
+DEBUG:  predicate pruning for shardId 630001
```

Comment:
This is due to shard pruning run twice for non-router plannable queries. It is really annoying me.  5.3 should fix this.